src/tetris/field/field.ts

```diff
@@ -66,24 +76,34 @@ export default class Field {
 	private _activeBrick: Brick;
 	private readonly _width: integer;
 	private readonly _height: integer;
+	private readonly _drawOffset: Vector2;
 	private readonly _brickFactory: BrickFactory;
 
+	private _nextActiveBrickDrop: number;
+	private _activeBrickDropInterval: number = 400;
+
 	// contains only stuck bricks
 	private readonly _bricks: Brick[];
 	private readonly _state: boolean[][];
 	//endregion
 
 	//region private methods
+	private _generateNewBrick(time: number): void {
+		this.activeBrick = this._brickFactory.newBrick(this);
+		this.activeBrick.field = this;
```

Comment:
The brickFactory should be responsible for correctly initializing a brick. This includes setting the field